### ISSUE TITLE
add key shortcuts for zoom changes

### ISSUE THREAD

**Issue by brainchild0** [marktext/marktext#2240]
Options *Zoom In* and *Zoom Out* have no keyboard shortcuts, though in applications such actions are commonly given some keyboard shortcut, often <kbd>Ctrl</kbd>+<kbd>+</kbd> (and <kbd>Ctrl</kbd>+<kbd>=</kbd>) for increasing level, and <kbd>Ctrl</kbd>+<kbd>-</kbd> for decreasing level.

**Comment by brainchild0** [comment]
Loosely related to  #1906.

**Comment by philip-n** [comment]
Just wanted to add a big :+1: for hotkey / mouse scroll support.

Background:

I like to use Marktext as a companion to [Joplin](https://joplinapp.org/). Joplin offers great markdown-based note organization / synchronization, and Marktext offers a clean interface, allowing to easily present notes from joplin e.g. during a meeting.

However, in a recent meeting, there was a 4K tv on which Marktext rendered the window font ineligibly small. I have to admit that after trying the common shortcuts (`Ctrl` + `+` / `Ctrl` + `mouse scroll`), I simply gave up and went another route.

*tl;dr: From a users perspective, the unavailability of the common shortcuts gave me the impression that zooming was simply not available in Marktext.*

**Comment by loyukfai** [comment]
CTRL +
CTRL + SHIFT +
CTRL + mouse scrool

Nice!

**Comment by loyukfai** [comment]
BTW, app doesn't remember the last zoom level currently.

**Comment by brainchild0** [comment]
> BTW, app doesn't remember the last zoom level currently.

Seems to be related to #1274.

**Comment by loyukfai** [comment]
Thanks! It seems the dev has gone dormant on this...?

**Comment by brainchild0** [comment]
I believe currently the project is receiving very little active maintenance.
